docs/gen2_Tutorial.md

```diff
@@ -358,6 +356,8 @@ arlon basecluster validategit --repo-alias prod --repo-path <pathToDirectory> [-
 
 ## gen2 cluster creation
 
+**Note: Base clusters only support dynamic profiles.**
```

Comment:
Does this cover documentation for CAS for gen2 clusters? I only see a note moved within gen2_tutorial.md

Reply:
I haven’t yet pushed the commit with the changes for CAS in gen2 clusters....I'll add it soon